modules/33acrossBidAdapter.js

```diff
@@ -756,20 +762,32 @@ function _createSync({ siteId = 'zzz000000000003zzz', gdprConsent = {}, uspConse
     sync.url += `&gdpr=${Number(gdprApplies)}`;
   }
 
+  if (viewportWidth) {
```

Comment:
One edge case I'd like to point out for each these four conditions is that if the value is 0 then this will result in the value not being sent, which may mistakenly result in generating a gLexId where in fact an iLexId should be generated. So instead, you could have something like:

`if (viewportWidth || typeof viewportWidth === 'number')`

You could also use Prebid.js's utility function `isNumber` as shorthand for `typeof xyz === 'number'`.